lib/shower/slide.js

```diff
@@ -0,0 +1,59 @@
+import { EventTarget, readOnly } from '../utils';
+
+/**
+ * Slide class.
+ *
+ * @param {HTMLElement} element
+ * @param {object} [options]
+ */
+class Slide extends EventTarget {
+    constructor(element, options) {
+        readOnly(this, {
```

Comment:
Do you mind `writable: false` and `Object.freeze`?

If some plugin will attempt to modify, for example, Shower options on-fly, it will fail loudly.

Reply:
Alternative to `writable: false` is `this._private` property and a getter. I believe that writability (part of ES5 property descriptors) is easier to grasp.

@pepelsbey, WDYT?